bridges/otellogr/convert.go

```diff
@@ -4,6 +4,7 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
+// Package otellogr provides conversion functionality for logging.
```

Comment:
This is strange. This (and other log bridge) packages already have Go Doc Comments for their packages. Is it not a false positive? 

- https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/bridges/otellogr/logsink.go
- https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/bridges/otellogrus/hook.go
- https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/bridges/otelslog/handler.go
- https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/bridges/otelzap/core.go